app/models/conference.rb

```diff
@@ -73,6 +74,11 @@ class Conference < ActiveRecord::Base
   after_create :create_free_ticket
   after_update :delete_event_schedules
 
+  enum ticket_layout: {
```

Comment:
Why about writing this as array: 

http://edgeguides.rubyonrails.org/4_1_release_notes.html#active-record-enums